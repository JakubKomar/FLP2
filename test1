A B
C D
E C
B C
A C
A E
E 
A D
B D