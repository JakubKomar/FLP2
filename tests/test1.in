A B
C D
E C
B C
A C
A E
E D
A D
B D