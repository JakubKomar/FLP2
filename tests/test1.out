./flp22-log
A-B A-C B-D E-C E-D 
A-B A-D B-C E-C E-D 
A-B A-E B-C C-D E-D 
A-B A-E B-D C-D E-C 
A-C A-E B-C B-D E-D 
A-D A-E B-C B-D E-C 
